app/controllers/cuisines_controller.rb

```diff
@@ -1,66 +1,38 @@
 class CuisinesController < ApplicationController
 
-  before_action :find_related_restaurants, :only => [:delete, :destroy]
-  before_action :find_cuisine, :except => [:index, :new, :create]
-
-
   def index
-    @cuisines = Cuisine.sorted_by_name_asc
+    render json: Cuisine.sorted_by_name_asc
   end
 
-  def show; end
-
-  def new
-    @cuisine = Cuisine.new
+  def show
+    render json: Cuisine.find(params[:id])
+  rescue ActiveRecord::RecordNotFound => error
```

Comment:
remove this rescue and the code below it as rails does this for you automatically.